kernelci/runtime/kubernetes.py

```diff
@@ -22,18 +22,19 @@ class Kubernetes(Runtime):
 
     JOB_NAME_CHARACTERS = string.ascii_lowercase + string.digits
 
-    def generate(self, params, plan_config):
-        template = self._get_template(plan_config)
+    @classmethod
+    def _get_job_file_name(cls, params):
+        return '.'.join([params['k8s_job_name'], 'yaml'])
+
+    def generate(self, params, job_config):
+        template = self._get_template(job_config)
         job_name = '-'.join(['kci', params['node_id'], params['name'][:24]])
         safe_name = re.sub(r'[\:/_+=]', '-', job_name).lower()
         rand_sx = ''.join(random.sample(self.JOB_NAME_CHARACTERS, 8))
         k8s_job_name = '-'.join([safe_name[:(62 - len(rand_sx))], rand_sx])
```

Comment:
I'd consider extracting it to a separate function.

Reply:
That's unrelated to this PR. Feel free to send a change to rework this if you want.

Reply:
ack